components/coding/studentPortal/Layout.tsx

```diff
@@ -36,6 +41,28 @@ export const Layout: React.FC = ({ children }) => {
     fetchPolicy: "cache-and-network",
   });
 
+  const { data } = useQuery<FetchUserAssignmentSubmissionsDataResponse>(
+    FETCH_USER_ASSIGNMENT_SUBMISSIONS,
+    {
+      variables: {
+        user_id: user.uid,
+      },
+      onCompleted: (data) => {
+        if (data.user_assignment_submissions) {
+          let instructorReviewed = data.user_assignment_submissions.filter(
```

Comment:
you can declare this variable with const since you're not re-assigning the variable anywhere else